src/utils/enums.ts

```diff
@@ -9,10 +9,23 @@ export enum UnitType {
   COMPANY = 'company'
 }
 
+const unitTypeMap: Map<string, UnitType> = new Map([['division', UnitType.DIVISION], ['brigade', UnitType.BRIGADE], ['battalion', UnitType.BATTALION], ['company', UnitType.COMPANY]]);
+
+export function parseUnitType(unitType: string) : UnitType {
+  return unitTypeMap.get(unitType)!;
+}
+
+const unitTypeLevelMap: Map<UnitType, number> = new Map([[UnitType.DIVISION, 3], [UnitType.BRIGADE, 2], [UnitType.BATTALION, 1], [UnitType.COMPANY, 0]]);
+
+export function unitTypeToLevel(unitType: UnitType) : number {
+  return unitTypeLevelMap.get(unitType)!;
+}
+
```

Comment:
These functions already exist in /utils/commons.

Reply:
Removed in commit `13616d`